### PR TITLE
Added note for Limitations

### PR DESCRIPTION
**Pull request by Melvin-CnC**
Added note about not sharing of monitors for the Limitations section.

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by Melvin-CnC** [comment]
@angadrandhawa1 Could you please review this change?

File: docs/alerts/monitors/index.md [hunk 1]
```diff
@@ -37,6 +37,10 @@ Notifications are optional and available as an **alert** and **recovery** for ea
 
 ## Limitations
 
+:::note
+You cannot share monitors, you must duplicate the monitor under another user.
```
Comment (JainM6) [inline docs/alerts/monitors/index.md:41]: I'd rephrase as:
You cannot share monitors; instead, you must duplicate the monitor under another user.

Comment (Melvin-CnC) [reply docs/alerts/monitors/index.md:41]: Thanks Megha! Updating the sentence, now.

Comment (stacykor) [reply docs/alerts/monitors/index.md:41]: Hey @Melvin-CnC you can update the changes right on this page https://github.com/SumoLogic/sumologic-documentation/pull/1487/files
Screenshot 2023-01-27 at 8 10 13 AM

**Comment by kimsauce** [comment]
Looks like this was merged in a different, new PR

File: docs/alerts/monitors/index.md (hunk 1 above)
Comment (kimsauce) [reply docs/alerts/monitors/index.md:41]: Was this change signed off by the PM @Melvin-CnC?
